### PR TITLE
Aj page

### PR DESCRIPTION
**Pull request by AllyJo**
Not really sure how the code review request works... added Peter & Gilad as reviewers.

### CODE REVIEW

File: dive-spots.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Places to Scuba Dive</title>
+  </head>
+  <body>
+    <h1>Dive Spots</h1>
+    <p>There are many great spots to scuba dive. If it's a source of water; mostly likely, you will be able to submerge yourself under meters of water.</p>
```
Comment (wolfram74) [inline dive-spots.html:8]: You might consider splitting this line up for legibility even though it doesn't have any child tags. 

\<p\>
  text
\</p\>

1: it's kind of long to be just one line
2: if in the future you do decide to put child elements in there, it'll already be formatted for legibility.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    <h2>Want to know more?</h2>
+    <p>Click the links below</p>
+
+    <a href="equipment.html">Equipment</a>
+    <a href="dive-spots.html">Dive Spots</a>
+
+    <h2>Existing Website Information</h2>
+    <p>I chose the <a href="https://www.padi.com/">Padi Website</a>
+    The website has a lot more happening than mine, which is to be expected. There are 'button' features that allow the user to navigate to other linked pages. The page also has click-able images, which will navigate to another page. Once I gain more experience with the different tags, (such as 'div', 'section' and 'class') and incorporating different languages (such as JSON and Javascript).  </p>
```
Comment (wolfram74) [inline index.html:36]: Cautionary note: JSON isn't so much a language as a way of formatting data.

**Comment by giladshanan** [comment]
Looks like great work to me!  It is easy to see that you worked hard on this.  Thanks for requesting my review, good to see how that looks on both sides.  The photos are great and it's nice that you include alt text, links to the originals as well as giving proper photo credits.

File: index.html (hunk 2 above)
Comment (giladshanan) [reply index.html:36]: Is 'class' an HTML tag or an attribute?

**Pull request merged**